django/contrib/auth/tests/custom_user.py

```diff
@@ -221,6 +223,11 @@ class CustomUserBadRequiredFields(AbstractBaseUser):
     class Meta:
         app_label = 'auth'
 
+
+class CustomUserUUIDasPK(AbstractUser):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
`default=uuid.uuid4,` is generally the correct way to set the default for a uuid field. It will get called automatically whenever a new instance is created.